readme.md

```diff
@@ -23,31 +50,34 @@ import { Widget } from "@uploadcare/react-widget";
 <Widget publicKey="demopublickey" />;
 ```
 
-## Available props
+## Configuration
 
-All the props except for the listed below are passed down to the widget. See [widget documentation](https://uploadcare.com/docs/file_uploads/widget/options/#options) for the details.
+### Component configuration
 
-### `value`: `string`
+#### `value`: `string`
 
-Set a file/group UUID or a [CDN link](https://uploadcare.com/docs/delivery/) as a value.
+Set a file/group UUID or a [CDN URL][delivery-docs] as a value.
 
 ```jsx
 <Widget value='9dd2f080-cc52-442d-aa06-1d9eec7f40d1' />
 <Widget value='9dd2f080-cc52-442d-aa06-1d9eec7f40d1~12' />
 <Widget value={null} />
 ```
 
-### `onFileSelect`: `(fileInfo: FileInfo) => void`
+#### `onFileSelect`: `(fileInfo: FileInfo) => void`
 
-### `onChange`: `(fileInfo: FileInfo) => void`
+#### `onChange`: `(fileInfo: FileInfo) => void`
 
-`FileInfo` object is described [here](https://uploadcare.com/docs/api_reference/rest/accessing_files/#properties)
+Set callbacks for the respective events. The `FileInfo` object is 
+described [here][api-refs-props].
 
-### `customTabs`: `{[string]: CustomTabConstructor}`
+#### `customTabs`: `{[string]: CustomTabConstructor}`
 
-Set a [custom tabs](https://uploadcare.com/docs/api_reference/javascript/custom_tabs/) for a widget.
+Set [custom tabs][custom-tabs-docs] for a widget.
```

Comment:
I'd say we add custom tabs rather than set them. Also, no notes on whether I need those at all.